easybuild/easyconfigs/p/Porechop/Porechop-0.2.4-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'Porechop'
+version = '0.2.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/rrwick/Porechop'
+description = """Porechop is a tool for finding and removing adapters from Oxford Nanopore reads.
+ Adapters on the ends of reads are trimmed off, and when a read has an adapter in its middle,
+ it is treated as chimeric and chopped into separate reads. Porechop performs thorough alignments
+ to effectively find adapters, even at low sequence identity"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/rrwick/Porechop/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['44b499157d933be43f702cec198d1d693dcb9276e3c545669be63c2612493299']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = ['none']
+use_pip = ['none']
```

Comment:
@sassy-crick This should be

```python
download_dep_fail = True
use_pip = True
```

I guess I should blame myself for this one, for not having written up proper documentation on all this...

Reply:
@boegel no worries. I am not always the best in reading and understanding them... :-)